server/config/client.ts

```diff
@@ -0,0 +1,40 @@
+import server, { ServerConfig } from './server'
+
+export type ClientConfig = Pick<ServerConfig,
+  'meta' |
+  'homeContent' |
+  'sponsors' |
+  'globalSiteTag' |
+  'ctfName' |
+  'divisions' |
+  'defaultDivision' |
+  'origin' |
+  'startTime' |
+  'endTime' |
+  'userMembers'
+> & {
+  emailEnabled: boolean;
+  ctftime: {
```

Comment:
Could be implemented as `Pick<ServerConfig['ctftime'], 'clientId'>`

Also missing nullable